abaplint.json

```
@@ -215,9 +215,9 @@
       "ignoreExceptions": true
     },
     "local_class_naming": {
-      "exception": "^LCL_.*$",
+      "exception": "^LCX_.*$",
       "local": "^LCL_.*$",
-      "test": "^LTCL_.*$"
+      "test": "^LT(C|D|H|S|CL)_.*$"
```

as suggested by Marc,

```suggestion
      "test": "^LT.+$"
```